engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
+  .cads-sidebar__content
+    %p.cads-sidebar__title= title
+    - if section_title.url.present?
+      %h3.cads-sidebar__section-title.cads-sidebar__section-title--small
+        = link_to section_title.title, section_title.url
+    - else
+      %h2.cads-sidebar__section-title
+        = section_title.title
```

Comment:
Kinda curious about this part. We make the title a h3 and smaller if it's linked or a h2 if not? Am I misreading?

Reply:
See: https://github.com/citizensadvice/design-system/pull/1693/files#r765567643

I think we can just have the `h3` regardless of whether there it contains a link or not